### ISSUE TITLE
Disable warnings for links to non-existing local files

### ISSUE THREAD

**Issue by clenk** [spatialaudio/nbsphinx#130]
**tldr; Please support links to sub-sections in non-notebook files.**

I'm using nbsphinx with autodoc and autosummary and would like to have a link from one of the notebooks to one of the pages that will be generated by autodoc. In a markdown cell in the notebook I use a link like `[something](../api/something.html#heading)`. In the final output the link works fine, but during the build process sphinx complains `WARNING file not found: 'api/something.html#heading'`.

After reading [this page in your docs](http://nbsphinx.readthedocs.io/en/0.2.14/markdown-cells.html#Links-to-*.rst-Files-(and-Other-Sphinx-Source-Files)), I added '.html' to `source_suffix` in my sphinx configuration. It doesn't throw a warning if I remove the anchor part of the link, but I need to be able to link to a specific section on the page.

The link works fine in the final output, but I want to use a lot of these links and I'm afraid sphinx dumping a ton of these warnings will make it harder to notice more important errors.

**Comment by clenk** [comment]
The fix in #129 works for me. I hope the LaTeX issue can be resolved and the PR merged in soon.

**Comment by mgeier** [comment]
It wouldn't be a problem to add a configuration option for suppressing those warnings.
But I've just seen that Sphinx recently introduced their own system for suppressing warnings:
https://github.com/sphinx-doc/sphinx/pull/2346
http://www.sphinx-doc.org/en/stable/config.html#confval-suppress_warnings

What about just adding `type` and `subtype` to all `nbsphinx` warnings?

BTW, you should not add `.html` to `source_suffix`, that doesn't make sense.

**Comment by clenk** [comment]
Yes, adding `type` and `subtype` to `nbsphinx` warnings seems like a better way to go.

Adding `.html` felt weird but I was just trying stuff out to see what would work.

**Comment by mgeier** [comment]
@clenk Would you like to make a PR for that?

**Comment by clenk** [comment]
Unfortunately It's unlikely I will have time to, and #135 solved my problem.

**Comment by mgeier** [comment]
@clenk No problem.

I think this feature has been added in Sphinx 1.4 and the documentation says "Now, this option should be considered *experimental*", so we should probably wait a bit anyway before implementing this.

I'm leaving this issue open for now ...

**Comment by mgeier** [comment]
I think this can be closed.

If someone still has a problem with those warnings, please leave a comment here!
